server/routes/applications.js

```diff
@@ -155,12 +159,48 @@ router.get('/manage-applications/:appId/delete', (req, res, next) => {
   res.render(`${routeViews}/index.njk`);
 });
 
-router.get('/manage-applications/:appId/api-key/add', (req, res, next) => {
-  logger.info(`GET request to serve index page: ${req.path}`);
-  const viewData = routeUtils.createViewData('Add Key', 'application-overview');
```

Comment:
I would still use the createViewData method in the routes. You can then modify the viewData variable and add in your additional fields.